site/content/docs/5.2/components/progress.md

```diff
@@ -87,6 +101,13 @@ Boosted also provides size variants for progress bar: simply add `.progress-xs`
 
 Use background utility classes to change the appearance of individual progress bars.
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
+These backgrounds color variants should not be used because they do not respect the Orange Design System specifications. The only background color to use is #ff7900.
```

Comment:
```suggestion
These backgrounds color variants should not be used because they do not respect the Orange Design System specifications. The only background color to use is the primary color.
```